test/driver_sanity/kernel_compat/vm_provider/virtualbox/Vagrantfile

```
@@ -0,0 +1,20 @@
+Vagrant.configure("2") do |config|
+
+  config.vm.provider "virtualbox"
+  config.ssh.forward_agent = true
+
+  config.vm.define "centos7" do | m |
+    m.vm.box = "bento/centos-7"
+    m.vm.hostname = "centos7"
+    m.vbguest.auto_update = false
+    m.vm.network "private_network", ip: "192.168.56.25"
+  end
+
+  config.vm.define "ubuntu" do | m |
+    m.vm.box = "bento/ubuntu-20.04"
+    m.vm.hostname = "ubuntu"
+    m.vbguest.auto_update = false
+    m.vm.network "private_network", ip: "192.168.56.26"
```

Same as previous comment
```suggestion
    m.vm.network "private_network", ip: ENV["LIBS_UBUNTU2004_IP"] || "192.168.56.26"
```